de-lib/src/main/java/org/iplantc/de/client/models/diskResources/MetadataTemplateAttributeType.java

```diff
@@ -0,0 +1,24 @@
+package org.iplantc.de.client.models.diskResources;
+
+public enum MetadataTemplateAttributeType {
+    BOOLEAN("Boolean"),
+    ENUM("Enum"),
+    INTEGER("Integer"),
+    MULTILINE("Multiline Text"),
+    NUMBER("Number"),
+    OLS_ONTOLOGY_TERM("OLS Ontology Term"),
```

Comment:
In order for the new ComboBox to be used in a Metadata Template attribute field, the attribute's type must be this `OLS Ontology Term` type.

This type will be added to Belphegor and the metadata database in separate PRs.